numbagg/decorators.py

```diff
@@ -1,8 +1,10 @@
 import numbers
+
 import numba
 import numpy as np
+import pandas as pd
```

Comment:
This means we're adding a run-time dependency on pandas. This is (probably) fine, but we should update `setup.py`. Alternatively we might use add a local import, so it's only needed if factorization is required.